api/vitest.utils.ts

```diff
@@ -211,7 +211,7 @@ If you are seeing this error, the root cause is likely an error thrown in the be
   });
 }
 
-export const defaultUserId = '64c7810107dd4782d32baee7';
+export const defaultUserId = '5bd30e0f1caf6ac3ddddddb5';
```

Comment:
Why does this need to change?

Reply:
We need the actual user who has completed the prerequisites instead of a random one.

Reply:
> This also makes a slightly unrelated change by using the demo user _id for the api tests and mocks. This makes it possible for local testing with mock data.

Reply:
Thanks, that makes sense. Maybe leave a comment explaining which user this is?